Swiftfin tvOS/Views/SettingsView/MaximumBitrateSettingsView.swift

```diff
@@ -24,21 +24,19 @@ struct MaximumBitrateSettingsView: View {
                     .frame(maxWidth: 400)
             }
             .contentView {
+
                 Section {
```

Comment:
Since you're here, please use the other init for the header. Since the footer has a conditional, it does not need to be changed,

Reply:
For the Footers I am a little lost. Here is what I am seeing as options for Footer usage:

<img width="646" alt="Screenshot 2024-08-06 at 13 43 51" src="https://github.com/user-attachments/assets/55ec40f6-84a2-433a-99d0-ec5942510e5d">

The version I think you are looking for looks like this? But I am getting some syntax errors:

<img width="989" alt="Screenshot 2024-08-06 at 13 44 22" src="https://github.com/user-attachments/assets/6a143be5-b261-4125-9856-fd8cf262dc33">

The original version looks like this if we want to retain this. Otherwise, I am more than happy to change this to keep everything consistent!

<img width="617" alt="Screenshot 2024-08-06 at 13 44 50" src="https://github.com/user-attachments/assets/1f761b39-392d-48ea-a6b5-be3d4c418213">

Reply:
Ah I apologize as I must have been mixed up with the inits with footers. We can leave this as is.